packages/@expo/cli/src/inlineModules/generation.ts

```diff
@@ -0,0 +1,433 @@
+import { getConfig } from '@expo/config';
+import { getPbxproj } from '@expo/config-plugins/build/ios/utils/Xcodeproj';
```

Comment:
```suggestion
import { IOSConfig } from '@expo/config-plugins';

const getPbxproj = IOSConfig.XcodeUtils.getPbxproj;
```

we can use getPbxproj from IOSConfig without access to internal path